app/src/main/java/org/oppia/android/app/mydownloads/MyDownloadsActivity.kt

```diff
@@ -17,20 +19,30 @@ class MyDownloadsActivity : InjectableAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    myDownloadsActivityPresenter.handleOnCreate()
+    val isFromNavigationDrawer = intent.getBooleanExtra(
+      BOOL_IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY,
+      /* defaultValue= */ false
+    )
     internalProfileId = intent.getIntExtra(KEY_NAVIGATION_PROFILE_ID, -1)
+    myDownloadsActivityPresenter.handleOnCreate(internalProfileId, isFromNavigationDrawer)
+    title = getString(R.string.menu_my_downloads)
```

Comment:
If you will apply the change mentioned in manifest then mostly this line won't be needed.

Reply:
done